### ISSUE TITLE
Is there a way to query for the Person responsible for making a particular DatasetVersion "release"?

### ISSUE THREAD

**Issue by mih** [HumanBrainProject/fairgraph#56]
I am aware of custodians and authors, but neither entities are associated with the action of creating a new `DatasetVersion` record in the knowledge graph. Is there a way to query for this with `fairgraph`?

**Comment by apdavison** [comment]
No. This information is not part of the openMINDS schemas, and is not included in the "hidden" metadata (fields with the key "https://core.kg.ebrains.eu/vocab/meta" in the JSON document) either.

I'll ask @olinux and @zehl to chime in on this, maybe we could add it?

**Comment by mih** [comment]
Thanks for the feedback!

**Comment by lzehl** [comment]
@mih thanks for bringing this up. I think it would be a good idea to add more information on who is doing what on the curation side in the KG. Within the systems metadata of the KG we could add who last changed it, who created an instance, and who released it (if this is not already there). I leave the decision for this to @olinux though, what do you think @olinux?

**Comment by apdavison** [comment]
Closing this issue as out of scope for fairgraph. I think it would be a good idea to add this information to the KG--the openMINDS schemas support it--but that is a task for the curation pipeline.
